modules/mobile-agents/android/client/src/org/wso2/mdm/agent/services/Operation.java

```diff
@@ -581,14 +581,14 @@ public void createWebClip(String code, String data) throws AndroidAgentException
 		String title = null;
 
 		try {
-			JSONObject webClipData = new JSONObject(code);
+			JSONObject webClipData = new JSONObject(data);
 			appUrl = (String) webClipData.get(resources.getString(R.string.intent_extra_identity));
 			title = (String) webClipData.get(resources.getString(R.string.intent_extra_title));
 		} catch (JSONException e) {
 			throw new AndroidAgentException("Invalid JSON format.", e);
 		}
 
-		resultBuilder.build(data);
+		resultBuilder.build(code);
 
 		if(appUrl != null && title != null){
```

Comment:
Its already been fixed in [https://github.com/wso2/product-mdm/commit/b0c66506d24fb5288300bec4224f1eaa58c7b426]